test/algorithms/buffer/CMakeLists.txt

```diff
@@ -38,6 +37,17 @@ foreach(item IN ITEMS
   target_compile_definitions(${BOOST_GEOMETRY_UNIT_TEST_NAME} PRIVATE BOOST_GEOMETRY_TEST_ONLY_ONE_TYPE)
 endforeach()
 
+# Tests which take a long time and are not run by default.
+# Besides that, the buffer_countries test requires the WKT files.
+foreach(item IN ITEMS
+    buffer_countries
+    buffer_linestring
+    buffer_linestring_geo
```

Comment:
The `buffer_linestring_geo` was not yet in cmake lists.
It's added now but very slow. Likewise `buffer_linestring` is slow.
They are not run by default now.